homeworks/rostyslav.vyshemirskyi_Rooosss/PokemonProject/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  font-family: "Roboto", sans-serif;
```

Comment:
Hey, @Rooosss, please add a design color of the text (your black is not the design's black:))